ipfs-lite/ipfs_lite.proto

```diff
@@ -182,37 +214,29 @@ message HashOnReadResponse {
 service IpfsLite {
     rpc AddFile(stream AddFileRequest) returns (AddFileResponse) {}
 
-    rpc GetFile(GetFileRequest) returns (GetFileResponse) {}
-
-    // Session() returns a NodeGetter which provides Get and GetMany methods
-
-    rpc GetNode(GetNodeRequest) returns (GetNodeResponse) {}
-
-    rpc GetNodes(GetNodesRequest) returns (stream GetNodesResponse) {}
-
-    // Node provides a ResloveLink method and a Resolver prop that provides Resolve and Tree methods
+    rpc GetFile(GetFileRequest) returns (stream GetFileResponse) {}
 
-    rpc ResolveLink(ResolveLinkRequest) returns (ResolveLinkResponse) {}
+    rpc HasBlock(HasBlockRequest) returns (HasBlockResponse) {}
 
-    rpc Resolve(ResolveRequest) returns (ResolveResponse) {}
+    // DAGService
 
-    rpc Tree(TreeRequest) returns (TreeResponse) {}
+    rpc AddNode(AddNodeRequest) returns (AddNodeResponse) {}
 
-    // BlockStore() returns a blockstore.Blockstore that provides the following
```

Comment:
Removed all the extra BlockStore methods because they are mostly redundant to the Node methods and not the primary way to use ipfs-lite